README.md

```diff
@@ -4,10 +4,22 @@
 
 Henceforth Tic Tac Toe may be referred to as "TTT" (or "Triple T").
 
-## Minimum Requirements
+## Workflow Requirements
+- By Monday at 9:00am
+  - Fork this repository
+  - To your fork, push a "readme" file that contains [5 user stories](https://github.com/ga-dc/curriculum/tree/master/02-intermediate-front-end/process-stories-wireframes)
```

Comment:
Maybe say "include the 3 from your quiz" or something like that? Also, I could make the quiz ask for 5 for consistency...